CMakeLists.txt

```diff
@@ -43,7 +43,8 @@ set(BUILD_BITS 64 CACHE STRING "Build for 64 (Geode default) or 32 bit.")
 
 set(PRODUCT_VENDOR "Apache" CACHE STRING "Product vendor")
 set(PRODUCT_VENDOR_NAME "The Apache Software Foundation" CACHE STRING "Product vendor full legal name")
-set(PRODUCT_NAME "Geode Native" CACHE STRING "Product name")
+set(PRODUCT_BASE_NAME "Geode" CACHE STRING "Product base name")
+set(PRODUCT_NAME "${PRODUCT_BASE_NAME} Native" CACHE STRING "Product name")
```

Comment:
Looks like this was done to be able to string build vendor.basename...

Reply:
@pdxcodemonkey do you have any updates?

Reply:
Waiting for a response to a question about Copyright string etc. then I'll update...